src/pages/home/CategoryCard.jsx

```diff
@@ -0,0 +1,21 @@
+import { useNavigate } from "react-router-dom";
+import { useProducts } from "../../context/product-context";
+
+export function CategoryCard({value}){
+    const {categoryName,image} = value;
```

Comment:
Here also you can destructure the image too.

Reply:
Okay, will do it later